lib/modules/manager/circleci/readme.md

```diff
@@ -1,3 +1,29 @@
 The `circleci` manager extracts both `docker` as well as `orb` datasources from CircleCI config files.
 
 If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+
+## Private orbs
+
+To get private orbs working you should:
+
+1. Encrypt the ... API token with ...
```

Comment:
This line items needs to fixed so that it explains:

- what's the name of the API token to encrypt
- how to encrypt it so it's safe and matches Circle CI's expectations

Reply:
Isn't this something like "encrypt the token from circleci using the renovate encryption page"?

Reply:
```suggestion
1. Encrypt your CircleCI token with the [Renovate encryption page](https://app.renovatebot.com/encrypt)
```